main.go

```diff
@@ -65,11 +65,15 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
+	var adapterClientRequestQPS float64
+	var adapterClientRequestBurst int
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.Float64Var(&adapterClientRequestQPS, "kube-api-qps", 20.0, "Set the QPS rate for throttling requests sent to the apiserver")
```

Comment:
I didn't notice this before, could you use `Float32Var` instead of `Float64Var` and then casting to `float32`
```suggestion
	flag.Float32Var(&adapterClientRequestQPS, "kube-api-qps", 20.0, "Set the QPS rate for throttling requests sent to the apiserver")
```

Reply:
There is no Float32Var function in `flag` package, only Float64Var. Float32Var is in `spf13/pflag`.
Either we cast to float32 or we use `spf13/pflag` instead of `flag`.

Reply:
yeah, you are right
The point is that we are already using it in metrics server, I suggested it only for the sake of the consistency :)

Reply:
So, pflag or not pflag ? :)
(https://github.com/vflaux/keda/compare/d179d4ec..6171365f)

Reply:
TBH, as you prefer, both are okay for me, but anything we do, should be consistent, I mean, if you use pflag here, use in adapter/main.go and if you don't use here, there neither